CHANGELOG.md

```diff
@@ -13,6 +13,12 @@
 
 ### Performance
 
+## 29.6.0
```

Comment:
Don't add a version header, I'll do that when making a release. Just stick your new entry under `Features` at the top

Reply:
Fixed